sdkproject/Assets/Mapbox/Unity/MeshGeneration/Factories/LowPolyTerrainFactory.cs

```diff
@@ -0,0 +1,517 @@
+namespace Mapbox.Unity.MeshGeneration.Factories
+{
+	using System.Collections.Generic;
+	using UnityEngine;
+	using Mapbox.Map;
+	using Mapbox.Unity.MeshGeneration.Enums;
+	using Mapbox.Unity.MeshGeneration.Data;
+	using Utils;
+
+	/// <summary>
+	/// Uses Mapbox Terrain api and creates terrain meshes.
+	/// </summary>
+	[CreateAssetMenu(menuName = "Mapbox/Factories/Low Poly Terrain Factory")]
+	public class LowPolyTerrainFactory : AbstractTileFactory
+	{
+		[SerializeField]
+		private Material _baseMaterial;
+		[SerializeField]
+		private MapIdType _mapIdType;
+		[SerializeField]
+		private string _customMapId = "mapbox.terrain-rgb";
+		[SerializeField]
+		private string _mapId = "";
+		[SerializeField]
+		private float _heightModifier = 2f;
+		[SerializeField]
+		private int _sampleCount = 10;
+		[SerializeField]
+		private bool _addCollider = false;
+		[SerializeField]
+		private bool _addToLayer = false;
+		[SerializeField]
+		private int _layerId = 0;
+
```

Comment:
do you think this is necessary? I felt like relative height is only important for realistic views.